Using-Mastodon/Apps.md

```diff
@@ -2,131 +2,130 @@ List of apps
-|[Mastalab](https://play.google.com/store/apps/details?id=fr.gouv.etalab.mastodon)|<https://bitbucket.org/tom79/mastodon_etalab>|[@tschneider@mastodon.etalab.gouv.fr](https://mastodon.etalab.gouv.fr/@tschneider)|
-|[Subway Tooter](https://play.google.com/store/apps/details?id=jp.juggler.subwaytooter)|<https://github.com/tateisu/SubwayTooter>|[@tateisu@mastodon.juggler.jp](https://mastodon.juggler.jp/@tateisu)|
-|[Tusky](https://f-droid.org/repository/browse/?fdcategory=Internet&fdid=com.keylesspalace.tusky&fdpage=3&fdstyle=grid)|<https://github.com/Vavassor/Tusky>|[@ConnyDuck@mastodon.social](https://mastodon.social/users/ConnyDuck)|
-|[Twidere](https://github.com/TwidereProject/Twidere-Android#twidere-for-android)|<https://github.com/TwidereProject/Twidere-Android>|[@mariotaku@pawoo.net](https://pawoo.net/@mariotaku)|
-|[Mastobone](https://play.google.com/store/apps/details?id=com.mastobone) (alpha)|<https://github.com/deguchi/mastobone>|[@deguchi@bookn.me](https://bookn.me/@deguchi)|
-|[MouseApp](https://f-droid.org/packages/fr.xtof54.mousetodon/) (beta)|<https://github.com/cerisara/mousetodon>|[@cerisara@mastodon.etalab.gouv.fr](https://mastodon.etalab.gouv.fr/@cerisara)
-|[TootyFruity](https://play.google.com/store/apps/details?id=ch.kevinegli.tootyfruity221258) (beta)|<https://github.com/eggplantPrince/tootyFruity>|[@eggplant@mastodon.social](https://mastodon.social/users/eggplant)|
-|[dndkodon](https://play.google.com/store/apps/details?id=jp.coe.dndkodon)|N/A|Tsuyoshi Hyuga|
-|[Cosian](https://play.google.com/store/apps/details?id=com.yatsuhashi.crims0n)|N/A|083|
-|[MastoPane](https://play.google.com/store/apps/details?id=com.mastopane)|N/A|Panecraft|
-|[mastty](https://play.google.com/store/apps/details?id=com.sasaneko.mastodon)|N/A|White Cat Factory|
-|[Pawoo](https://play.google.com/store/apps/details?id=jp.pxv.pawoo)|N/A|[@pixiv@pawoo.net](https://pawoo.net/users/pixiv)|
-|[Stegodon](https://play.google.com/store/apps/details?id=com.gilawhost.stegolophodon)|N/A|[@GilaW@mastodon.cloud](https://mastodon.cloud/@Gilaw)|
-|[Tootaloo](https://play.google.com/store/apps/details?id=net.liverworks.tootalooa)|N/A|[liverworks.net](http://liverworks.net/)|
-|[Tootdon](https://play.google.com/store/apps/details?id=club.tootdon.app)|N/A/|[@tootdon@mstdn.jp](https://mstdn.jp/@tootdon)|
-|[Troutoss](https://play.google.com/store/apps/details?id=jp.gr.java_conf.miwax.troutoss)|N/A|[@tomoya0x00@mstdn.jp](https://mstdn.jp/@tomoya0x00)|
+|**11t**<br />([Play](https://play.google.com/store/apps/details?id=com.jeroensmeets.mastodon))|<https://github.com/jeroensmeets/mastodon-app>|[@jeroensmeets@mastodon.social](https://mastodon.social/users/jeroensmeets)|
+|**[AndStatus](http://andstatus.org/)**<br />([Play]()/[F‑Droid]())|<https://github.com/andstatus/andstatus>|[@AndStatus@mastodon.social](https://mastodon.social/@AndStatus)|
```

Comment:
If this one doesn't have Play/F-Droid links maybe you should leave them out?

Reply:
Also all the others have the links removed but this one is still bold and linked.